src/Common/tests/SystemEventsHelper.cs

```diff
@@ -0,0 +1,89 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Reflection;
+using Microsoft.Win32;
+using Xunit;
+using static Interop.User32;
+
+namespace WinForms.Common.Tests
+{
+    public static class SystemEventsHelper
+    {
+        private static IntPtr GetHWnd()
+        {
+            // locate the hwnd used by SystemEvents in this domain
+            FieldInfo windowClassNameField = typeof(SystemEvents).GetField("s_className", BindingFlags.Static | BindingFlags.NonPublic) ??  // corefx
+                                        typeof(SystemEvents).GetField("className", BindingFlags.Static | BindingFlags.NonPublic);      // desktop
+            Assert.NotNull(windowClassNameField);
+            string windowClassName = windowClassNameField.GetValue(null) as string;
+            Assert.NotNull(windowClassName);
+
+            IntPtr window = FindWindowW(windowClassName, null);
+            return window;
+        }
+
+        public static void CallUserPreferenceChanged(UserPreferenceCategory category)
```

Comment:
`Call` feels foreign to the WinForms vocabulary. We typiclly use `Invoke`, `Raise`, `Send` or `Post`.
Maybe something like this?
```suggestion
        public static void SendlMessageOnUserPreferenceChanged(UserPreferenceCategory category)
```